client/src/featureform/serving.py

```diff
@@ -15,90 +15,37 @@
-                                                                                 feature_column_name,
-                                                                                 dataframe_mapping,
-                                                                                 featureVariantTuple[0], "")
-        try:
-            all_feature_df = dataframe_mapping[feature_dataframes.pop()]
-            while len(feature_dataframes) != 0:
-                featureDF = dataframe_mapping[feature_dataframes.pop()]
-                all_feature_df = all_feature_df.join(featureDF.set_index(entity_tuple[0]), on=entity_tuple[0], lsuffix='_left')
-        except TypeError:
-            print("Set is empty")
-        entity_row = all_feature_df.loc[all_feature_df[entity_tuple[0]] == entity_tuple[1]]
-        return entity_row
-
-    def process_feature_csv(self, source_path, entity_name, feature_dataframes, feature_column_name, dataframe_mapping,
-                            featureName, timestamp_column):
-        df = pd.read_csv(str(source_path))
-        if timestamp_column != "":
-            df = df[[entity_name, feature_column_name, timestamp_column]]
+    def __init__(self, host=None, local=False, tls_verify=True, cert_path=None):
+        self.local = local
+        if local:
+            if host != None:
+                raise ValueError("Cannot be local and have a host")
+            self.sqldb = SQLiteMetadata()
         else:
-            df = df[[entity_name, feature_column_name]]
-
-        df.set_index(entity_name)
-        feature_dataframes.add(featureName)
-
-        dataframe_mapping[featureName] = df.sort_values(by=feature_column_name, ascending=False)
-        return feature_dataframes, dataframe_mapping
+            env_cert_path = os.getenv('FEATUREFORM_CERT')
+            if tls_verify:
+                credentials = grpc.ssl_channel_credentials()
+                channel = grpc.secure_channel(host, credentials)
+            elif cert_path is not None or env_cert_path is not None:
+                if env_cert_path is not None and cert_path is None:
+                    cert_path = env_cert_path
+                with open(cert_path, 'rb') as f:
+                    credentials = grpc.ssl_channel_credentials(f.read())
+                channel = grpc.secure_channel(host, credentials)
+            else:
+                channel = grpc.insecure_channel(host, options=(('grpc.enable_http_proxy', 0),))
+            self._stub = serving_pb2_grpc.FeatureStub(channel)
 
-    def processLabelCSV(self, source_path, entity_name, labelColumnName, timestamp_column):
-        df = pd.read_csv(source_path)
-        if timestamp_column != "":
-            df = df[[entity_name, labelColumnName, timestamp_column]]
-        else:
-            df = df[[entity_name, labelColumnName]]
-        df.set_index(entity_name)
-        return df
+    def dataset(self, name, version):
```

Comment:
I think this should probably be the same as training_set right? Since we want to APIs to look the same. @simba-git